backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,146 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor._
+import cromwell.backend.model.{JobDescriptor, WorkflowDescriptor}
+
+import scala.concurrent.Future
+import scala.util.{Success, Failure}
+
+object BackendActor {
+
+  // Commands
+  sealed trait BackendActorMessage
+  case class Execute(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case class Abort(jobDescriptor: JobDescriptor) extends BackendActorMessage
+  case object AbortAll extends BackendActorMessage
```

Comment:
although to play devils advocate how is everything aborted, perhpas with per-call abort signals internally